www/main/common/store/user-settings/get-saved-user-settings.js

```diff
@@ -20,7 +20,6 @@ function parseDataFile(filePath) {
     Object.keys(settings).forEach(key => {
       defaultSettings[key] = settings[key].initialValue;
     });
-    fs.writeFileSync(userConfigPath, JSON.stringify(defaultSettings));
```

Comment:
so we won't write this file until a setting is changed? should be fine...?

Reply:
Yes, that's how the store used to work previously.